### PR TITLE
Mustafe-Onboarding

### PR DESCRIPTION
**Pull request by Muste8**
### Changes
-Double checked that all the onboarding tasks on this readme
-I answered all of the check-in questions(including this one!) listed throughout this readme.
-I'm ready to submit this readme for review!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -108,7 +108,7 @@ either version manager installed, and you should install `rbenv`.
 **DO NOT INSTALL BOTH RVM AND RBENV ON THE SAME MACHINE OR YOU WILL HAVE A VERY BAD TIME!**
 
 *Check-in: I DO NOT have RVM and RBENV installed.* 
-- [ ] Not at all! 
+- [x] Not at all!
```
Comment (camckin10) [inline README.md:111]: Double checking that you were able to install rvm or rbenv on your computer. If thats ready, and there are no issues, please update your readme to reflect the change! :gratitude-thank-you:

Comment (Muste8) [reply README.md]: I just updated my readme. Thanks!

**Review by camckin10 — APPROVED** [review]
Looking good!

**Review by edithemmings — CHANGES_REQUESTED** [review]
Looks great! You're welcome to close this PR

**Pull request closed without merging**